### DISCUSSION TITLE
/protobuf/cmake/protoc-3.11.3.0: Exec format error

### DISCUSSION THREAD

**Discussion by JewelKing** [microsoft/onnxruntime#6858] [Other Q&A]
[ 29%] Running C++ protocol buffer compiler on /home/zmotion/Downloads/onnxruntime-1.6.0/build/Linux/Release/external/onnx/onnx/onnx-ml.proto
/bin/sh: 1: ../protobuf/cmake/protoc-3.11.3.0: Exec format error
external/onnx/CMakeFiles/gen_onnx_proto.dir/build.make:80: recipe for target 'external/onnx/onnx/onnx-ml.pb.cc' failed
make[2]: *** [external/onnx/onnx/onnx-ml.pb.cc] Error 2
CMakeFiles/Makefile2:2337: recipe for target 'external/onnx/CMakeFiles/gen_onnx_proto.dir/all' failed
make[1]: *** [external/onnx/CMakeFiles/gen_onnx_proto.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 29%] Built target pep8_check
Makefile:181: recipe for target 'all' failed

**Comment by jcwchen** [comment]
Hi @JewelKing,
Similar to this https://github.com/pranav-prakash/onnxruntime-riscv/issues/7? Could you try the solution there and what's your environment? Thanks.

**Comment by snnn** [comment]
Could you please upload the full build log here and let us know your build command and system environment info(OS version, compiler version,etc)?
